main/src/com/google/refine/operations/recon/ReconJudgeSimilarCellsOperation.java

```diff
@@ -222,10 +222,10 @@ public boolean visit(Project project, int rowIndex, Row row) {
                                     recon.judgment = Recon.Judgment.New;
                                     recon.match = null;
                                 } else if (_judgment == Judgment.None) {
-
-                                    recon.judgment = recon.error == null ? Recon.Judgment.None : Recon.Judgment.Error;
-
+                                    recon.judgment = Recon.Judgment.None;
```

Comment:
I think it is still worth having this logic here, so that if the user discards a match, we still set the judgment to "error" if the recon's error is not null. It's a different case, independent from the `else if` block you added below
```suggestion
                                    recon.judgment = recon.error == null ? Recon.Judgment.None : Recon.Judgment.Error;
```